app/src/main/java/net/aiscope/gdd_app/ui/metadata/MetadataActivity.kt

```diff
@@ -3,19 +3,22 @@ package net.aiscope.gdd_app.ui.metadata
 import android.content.Intent
 import android.os.Bundle
 import android.view.View
-import android.widget.AbsSpinner
 import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import androidx.recyclerview.widget.LinearLayoutManager
 import com.google.android.material.snackbar.Snackbar
 import dagger.android.AndroidInjection
-import kotlinx.android.synthetic.main.activity_metadata.*
-import kotlinx.android.synthetic.main.toolbar.*
+import kotlinx.android.synthetic.main.activity_metadata.metadata_blood_sample_images
```

Comment:
Not sure about these synthetic imports, do you feel that it's better without the star?
Btw, maybe you want to have a look on the new shiny thing, [View Binding](https://developer.android.com/topic/libraries/view-binding).

Reply:
Yes, normally I'd go without any star import, but I agree the synthetic ones are a different case. I had removed their aggregation on the projects config, but then I reverted that (kept the no-grouping for java.util, though). I just pushed a commit with imports reorganisation for the activities. Thanks for the View Binding reference - I think I like it, have to try! :-D